lib/rules/property-sort-order.js

```diff
@@ -143,5 +140,43 @@ module.exports = {
     });
 
     return result;
+  },
+  'fix': function (groot, parser) {
```

Comment:
Do not comma-separate off of `module.exports` for `'fix'`. Create a separate fix function and use `module.exports.fix = fix`